lib/org/chromium/webidl/Diff.js

```diff
@@ -276,19 +276,13 @@ foam.CLASS({
           var leftProps = left.cls_.getAxiomsByClass(foam.core.Property);
           var rightProps = right.cls_.getAxiomsByClass(foam.core.Property);
 
-          // Determine if left and right have the same properties.
-          // FUTURE: Relax condition and iterate over sorted collections of
-          // left and right properties.
-          if (foam.util.compare(leftProps, rightProps) !== 0) {
-            // Cannot perform a deep comparison of the objects.
-            this.addChunk_(id, this.DiffStatus.VALUES_DIFFER, left,
-                right, changes);
-            return;
-          }
+          // Perform a diff on all common properties.
```

Comment:
We verify that the two objects are the same type. Will objects of the same type potentially have different properties?